website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,22 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding an empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set.
```

Comment:
```suggestion
As filtering is reductive in nature and driven by a user action, it’s not always possible to avoid an empty state.
```

Reply:
I'm curious as to why you would want to "avoid" an empty state when filtering, though. If nothing is found for the filter you've set, then an empty state is valid.